### ISSUE TITLE
Playback Fails From v308 Onwards

### ISSUE THREAD

**Issue by mmanuxx** [bennettpeter/android-MythTV-Leanfront#27]
I'm using nvidia shield frontend and mythtv backend v0.28. Updating from v306 results in playback failure. I updated to v308 in December. When that failed, I reverted to v306. Tried again with v340. Same issues after updating. Episodes have no thumbnails, just the generic white icon. When attempting playback, it goes to fullscreen briefly then fails and returns to the prior screen. I cleared cache & data and retried to no avail. Also, I Iooked for v307 in the Prior Versions folder at the download site to see if that would work but didn't find it.

**Comment by bennettpeter** [comment]
That is when I added support for playback groups. I suppose that the playback group is not returned in the API in V0.28. I will take a look and see what can be done.

**Comment by bennettpeter** [comment]
Please provide the exact MythTV version. You can get that by running mythutil --version.

**Comment by mmanuxx** [comment]
mythutil version: fixes/0.28 [v0.28.1-12-g8f13742]

**Comment by bennettpeter** [comment]
I tried but I cannot build that version on my system. I would have to install a lot of out of date packages to do that.

Please install the latest version of leanfront 20220307-leanfront-release-v0-340-g4f4d93f. Open it and attempt playback. Then after the crash get a log file listing by running this:
adb logcat -d > tmp/android.log
Send me the log file.

**Comment by mmanuxx** [comment]
Not as familiar navigating within an android box as I am a PC. How do I get that log file? Where do I input that command? Do I install an android terminal app and issue that command while mythtv leanfront is open? Is there a place within the leanfront app to issue that command?

**Comment by bennettpeter** [comment]
Install adb on Linux (you probably already installed that to use when installing leanfront?).
Enable debugging on android (you probably did that to install leanfront with adb?).
connect adb by running this from linux:
```
adb connect <android ip address>

```
Run leanfront on android to cause the crash. Immediately after the crash, run this on linux:
```
adb logcat -d > tmp/android.log
adb disconnect

```

**Comment by mmanuxx** [comment]
Only interacted with nvidia shield directly(not remotely) using a keyboard remote with mouse function and downloading and installing apks including leanfront via web browser on the shield. Anyway I enabled USB network debuging on the shield, installed adb on my linux laptop, reinstalled leanfront 340. But I'm stuck at this point below after attempted playback:

`user@xxxx:~$ adb connect 10.0.0.95:5555`
`connected to 10.0.0.95:5555`
`user@xxxx:~$ adb logcat -d > tmp/android.log`
`bash: tmp/android.log: No such file or directory`

**Comment by bennettpeter** [comment]
That assumed you have a directory called tmp. If not, just run it this way
```
adb logcat -d > android.log
adb disconnect

```
You will have to redo the test from the beginning as the log is not kept for long on the android.
Then send me the android.log.

**Comment by mmanuxx** [comment]
It appears to create it after issuing the command `adb logcat -d > android.log` since it just returns to the next line in the terminal, but I can't find the android.log file anywhere when searching via an android file explorer app. Searched google for android log file locations but there was no clarity there. Stuck.

**Comment by bennettpeter** [comment]
This is elementary linux shell stuff. The redirect operator > outputs to a file. If you specify the file name without a directory (e.g. android.log), it creates it in the current directory. That will probably be your linux home directory. So the android.log is likely in your linux home directory.

https://ubuntu.com/tutorials/command-line-for-beginners#4-creating-folders-and-files

**Comment by mmanuxx** [comment]
[android2104.log](https://github.com/bennettpeter/android-MythTV-Leanfront/files/8236852/android2104.log)

**Comment by bennettpeter** [comment]
Thanks, that was very helpful. I believe I know what the problem is. I have built a new version. Please test it and let me know if it fixes your bug. It is version 20220312-leanfront-trial-v0-341-g362514d.apk in https://dl.orangedox.com/pCBmBm under "Trial Versions"

**Comment by mmanuxx** [comment]
Success! Plays without issue and thumbnails have returned. Thanks for your work.

**Comment by bennettpeter** [comment]
Fixed in 362514d
